### PR TITLE
CNV-31882: Move common instance types to virt operator

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/CNV-31882

CNV 4.16+

Preview:
https://75343--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/about_virt/virt-architecture.html#virt-about-virt-operator_virt-architecture
https://75343--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/about_virt/virt-architecture.html#virt-about-ssp-operator_virt-architecture

### CODE REVIEW

File: modules/virt-about-ssp-operator.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="virt-about-ssp-operator_{context}"]
 = About the Scheduling, Scale, and Performance (SSP) Operator
 
-The SSP Operator, `ssp-operator`, deploys the common templates, the related default boot sources, the pipeline tasks, and the template validator.
+The SSP Operator, `ssp-operator`, deploys the default boot sources, the pipeline tasks, and the template validator.
```
Comment (dominikholler) [inline modules/virt-about-ssp-operator.adoc:10]:
```suggestion
The SSP Operator, `ssp-operator`, deploys the common templates, the related default boot sources, and the template validator.
```
`common templates` and `common instancetypes` are two different things.

The `common templates` stays in ssp-operator. Looks like the `common instancetypes` was not yet documented here.

The ` the pipeline tasks` should be removed, too. https://issues.redhat.com/browse/CNV-29468

File: modules/virt-about-virt-operator.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [id="virt-about-virt-operator_{context}"]
 = About the {VirtProductName} Operator
 
-The {VirtProductName} Operator, `virt-operator` deploys, upgrades, and manages {VirtProductName} without disrupting current virtual machine (VM) workloads.
+The {VirtProductName} Operator, `virt-operator`, deploys, upgrades, and manages {VirtProductName} without disrupting current virtual machine (VM) workloads. In addition, the {VirtProductName} Operator deploys the common VM templates.
```
Comment (dominikholler) [inline modules/virt-about-virt-operator.adoc:10]:
```suggestion
The {VirtProductName} Operator, `virt-operator`, deploys, upgrades, and manages {VirtProductName} without disrupting current virtual machine (VM) workloads. In addition, the {VirtProductName} Operator deploys the common instance types and common preferences.
```

**Review by dominikholler — CHANGES_REQUESTED** [review]
@lyarwood

**Comment by geetikakay** [comment]
cc @RoniKishner  please review

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by jeana-redhat** [comment]
@sbeskin-redhat just verifying, should "CNV 4.16" be interpreted as OCP versions 4.16+?

**Comment by sbeskin-redhat** [comment]
@jeana-redhat  Yes, I've updated the description. Thank you

**Pull request merged**